docs/docs/how-to/file-uploads.md

```diff
@@ -446,6 +446,50 @@ Now let's add the ability to bring back the uploader if you decide you want to c
 
 We're borrowing the styles from the submit button and making sure that the image has both a top and bottom margin so it doesn't crash into the new button.
 
+## Safe usage of your free plan
+Having a free plan is great, but if you just load images and never actually remove the unnecessary ones, you'll be in trouble.
+
+To avoid this, we'd better implement the `deleteImage` mutation. It will enable you to make a call to the Filestack API to remove your resources, and on success, you will remove the row in the `Image` model.
+
+You are going to need a new `.env` called `REDWOOD_ENV_FILESTACK_SECRET`, which you can find in Filestack > Security > Policy & Signature: App Secret.
+
+Also, let's add this package to the right side of our current concern:
+
+```shell
+yarn workspace api add filestack-js
+```
+
+You should be good to go. Let's modify our service accordingly:
+
+```js {4-23} title="api/src/services/image/image.ts"
+import * as Filestack from 'filestack-js'
+
+export const deleteImage = async({ id }) => {
+  const client = Filestack.init(process.env.REDWOOD_ENV_FILESTACK_API_KEY)
+
+  const image = await db.image.find({ where: { id } })
+
+  /** @manual the `security.handle` is the unique part of the Filestack file's url. */
+  const handle = /\w+$/.exec(image.url).pop()
+  
+  const security = Filestack.getSecurity(
+    {
+      /** @manual We set `expiry` at `now() + 5 minutes`. */
+      expiry: new Date(new Date().getTime() + 5 * 60 * 1000),
+      handle,
+      call: ['remove'],
+    },
+    process.env.REDWOOD_ENV_FILESTACK_SECRET
+  )
+
+  return client
+    .remove(handle, security)
+    .then(async (body) => db.image.delete({ where: { id } } ))
```

Comment:
You've got the `deleteImage()` function denoted as `async` but then use Promise syntax here. Personally I prefer async/await in code examples, could this become:

```js
  await client.remove(handle, security)
  return db.image.delete({ where: { id }})
```